### PR TITLE
delete unnecessary files

### PR DESCRIPTION
**Pull request by ozizhakGT**

### CODE REVIEW

**Review by anniebelkin — CHANGES_REQUESTED** [review]
You should remove the whole .idea folder and add it to gitignore

**Comment by ozizhakGT** [comment]
> You should remove the whole .idea folder and add it to gitignore

**Comment by Avishayy** [comment]
This was not merged, I'm reopening

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+/idea
```
Comment (anniebelkin) [inline .gitignore:7]: change to /.idea

**Review by anniebelkin — CHANGES_REQUESTED** [review]
remove .idea folder from repository

**Review by anniebelkin — APPROVED** [review]
:)

**Pull request merged**